packages/devtools_app/lib/src/main.dart

```diff
@@ -80,6 +82,13 @@ class HtmlPerfToolFramework extends HtmlFramework {
           .open('https://github.com/flutter/devtools/issues', '_feedback');
     });
 
+    if (enableFlutterWebTesting) {
+      // Listen for clicks on the 'Try DevTools on Flutter Web' button.
+      queryId('try-flutter-web-devtools').onClick.listen((_) {
+        // TODO(kenz): launch flutter web app.
+      });
+    }
```

Comment:
we need to hide the icon when we aren't really supporting it.

Reply:
whoops, good catch